mcodingbot/utils/peps.py

```diff
@@ -71,5 +96,16 @@ def embed(self) -> hikari.Embed:
             color=CONFIG.theme,
         ).set_author(name=self.authors)
 
+    def truncated_title(self) -> str:
+        pep_digits = len(str(self.number))
+
+        max_name_length = 100 - pep_digits - 3
+
+        name = self.title
+        if len(name) > max_name_length:
+            name = f"{self.title[:max_name_length - 3]}..."
+
+        return f"{name} ({self.number})"
+
```

Comment:
I see `-3` twice... Also 100 looks like a magic number.

At least some comments would be nice.

Reply:
True it is a magic number, ill add some comments.